specs/darwin/kernel_panics.table

```diff
@@ -0,0 +1,18 @@
+table_name("kernel_panics")
+description("Kernel Panics")
+schema([
+    Column("path", TEXT, "Location of log file"),
+    Column("time", TEXT, "Formatted time of the event"),
+    Column("registers", TEXT, "A space delimited line of register:value pairs"),
+    Column("frame_backtrace", TEXT, "Backtrace of the crashed module"),
+    Column("module_backtrace", TEXT, "Modules appearing in the crashed module's backtrace"),
+    Column("module_dependency", TEXT, "Module dependencies existing in crashed module's backtrace"),
+    Column("name", TEXT, "Process name corresponding to crashed thread"),
+    Column("os_version", TEXT, "Version of the operating system"),
+    Column("kernel_version", TEXT, "Version of the system kernel"),
+    Column("system_model", TEXT, "Physical system model"),
```

Comment:
Can you provide an example of the output here? I'm looking for opportunities to normalize the column names with existing tables like `os_version` and `system_info`.